content/ch-algorithms/teleportation.ipynb

```diff
@@ -22415,14 +22415,82 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 4. Teleportation on a Real Quantum Computer <a id='real_qc'></a>"
+    "## 4. Understanding Quantum Teleportation "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As you have worked with the Quantum Teleportation's implementation, it is time to understand the mathematics behind the protocol.\n",
+    "\n",
+    "#### Step 1\n",
+    "\n",
+    "Quantum Teleportation begins with the fact that Alice needs to transmit $|q\\rangle = a|0\\rangle + b|1\\rangle$ (a random qubit) to Bob. She doesn't know the state of the qubit. For this, Alice and Bob take the help of a third party (Telamon). Telamon prepares a pair of entangled qubits for Alice and Bob. The entangled qubit could be written in Dirac Notation as,\n",
+    "$$ |\\psi \\rangle = \\frac{1}{\\sqrt{2}} (|00\\rangle + |11\\rangle) $$\n",
+    "\n",
+    "Alice and Bob each possess one qubit of the entangled pair (denoted as A and B respectively),\n",
+    "$$|\\psi\\rangle = \\frac{1}{\\sqrt{2}} (|0\\rangle_A |0\\rangle_B + |1\\rangle_A |1\\rangle_B) $$\n",
+    "\n",
+    "This creates a three qubit quantum system where Alice has the first two qubits and Bob the last one.\n",
+    "$$ \\begin{align}\n",
+    "|q\\rangle \\otimes |\\psi\\rangle &= \\frac{1}{\\sqrt{2}} (a |0\\rangle \\otimes (|00\\rangle + |11\\rangle) + b |1\\rangle \\otimes (|00\\rangle + |11\\rangle))\\\\\n",
+    "&= \\frac{1}{\\sqrt{2}} (a|000\\rangle + a|011\\rangle + b|100\\rangle + b|111\\rangle) \n",
+    "\\end{align}$$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Step 2\n",
+    "\n",
+    "Now according to the protocol Alice applies CNOT gate on her two qubits followed by Hadamard gate on the first qubit. This results in the state,\n",
+    "\n",
+    "$$ \\begin{align} (CNOT \\otimes I) (H \\otimes I \\otimes I) (|q\\rangle \\otimes |\\psi\\rangle) \\\\\n",
+    "&=(CNOT \\otimes I) (H \\otimes I \\otimes I) \\frac{1}{\\sqrt{2}} (a|000\\rangle + a|011\\rangle + b|100\\rangle + b|111\\rangle) \\\\\n",
+    "&= (H \\otimes I) \\frac{1}{\\sqrt{2}} (a|000\\rangle + a|011\\rangle + b|110\\rangle + b|101\\rangle) \\\\\n",
+    "&= \\frac{1}{2}  (a(|000\\rangle + |011\\rangle + |100\\rangle + |111\\rangle) + b(|010\\rangle + |001\\rangle - |110\\rangle - |101\\rangle)) \\\\\n",
+    "&= \\frac{1}{2} ( |00\\rangle (a|0\\rangle + b|1\\rangle) + |01\\rangle (a|1\\rangle + b|0\\rangle) +|10\\rangle (a|0\\rangle - b|1\\rangle) +|11\\rangle (a|1\\rangle - b|0\\rangle) )\n",
+    "\\end{align}$$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Step 3\n",
+    "\n",
+    "Alice measures the first two qubit (which she owns) and sends them as two classical bits to Bob. The result she obtains is always one of the four standard basis states $|00\\rangle, |01\\rangle, |10\\rangle,$ and $|11\\rangle$ with equal probability.  \n",
```

Comment:
Related to[ this suggestion](https://github.com/qiskit-community/qiskit-textbook/pull/651/files#r469926240), it would be cool to see a table-like equation here showing:
```
|00>  --->  a|0> + b|1>
|01>  --->  b|1> + a|0> 
etc.
```
[Similar to this equation here](https://qiskit.org/textbook/ch-algorithms/teleportation.html#Step-4).

Reply:
Yes, I'm working on this only currently. :)